mqtt-cs/src/main/java/org/apache/rocketmq/mqtt/cs/session/infly/InFlyCache.java

```diff
@@ -61,9 +61,7 @@ public void cleanResource(String clientId,String channelId) {
 
     public void put(CacheType cacheType, String channelId, int mqttMsgId) {
         ConcurrentMap<String, Set<Integer>> cache = whichCache(cacheType);
-        if (!cache.containsKey(channelId)) {
```

Comment:
Adding judgment is mainly to avoid unnecessary temporary object generation

Reply:
Oh, that's my mistake. I just replaced them all, and don't notice this cache operation will be frequent.